app/lib/LoggingFilter.scala

```diff
@@ -12,9 +13,9 @@ class LoggingFilter(override val mat: Materializer) extends Filter with Logging
     val ec = scala.concurrent.ExecutionContext.Implicits.global
 
     val startTime = System.currentTimeMillis
-    val headers = requestHeader.headers.getAll(LoggingContext.LOGGING_CONTEXT_HEADER)
-      .map(LoggingContext.fromHeader)
```

Comment:
Are we okay not to pass the context header through this function any more? It looks like it's doing some meaningful parsing of the header

Reply:
fair point, re-instated